src/Methods/Runes/GetAddressRuneUTXOIndex.ts

```diff
@@ -1,19 +1,25 @@
-import { method, NotFoundError } from "@valkyr/api";
+import { method } from "@valkyr/api";
 import Schema, { string } from "computed-types";
 import { RuneUtxoBalance } from "runestone-lib";
 
 import { runes } from "~Database/Runes";
+import { FindPaginatedParams } from "~Libraries/Paginate";
+
+import { schema } from "../../Libraries/Schema";
 
 export default method({
   params: Schema({
     address: string,
     runeTicker: string,
+    pagination: schema.pagination.optional(),
   }),
-  handler: async ({ address, runeTicker }): Promise<RuneUtxoBalance[]> => {
-    const balances = await runes.addressRunesUTXOs(address, runeTicker);
-    if (balances === null) {
-      throw new NotFoundError("Address not found");
-    }
-    return balances;
+  handler: async ({ address, runeTicker, pagination = {} }) => {
+    const params: FindPaginatedParams<RuneUtxoBalance> = { ...pagination, filter: { address, runeTicker } };
```

Comment:
Should we add a default sorting for pagination? I think without a sort key, the ordering of the results returned by MongoDB is not stable